<?xml version="1.0"?>
<launch>
    <arg name="robot_ip" default="172.18.1.2" />
    <arg name="pose_name" />

    <include file="$(find franka_control)/launch/franka_control.launch">
        <arg name="robot_ip" value="$(arg robot_ip)" />
        <arg name="load_gripper" value="false" />
    </include>

    <param name="moveit_controller_manager" value="moveit_ros_control_interface::MoveItControllerManager" />
    <include file="$(find panda_moveit_config)/launch/panda_moveit.launch">
        <arg name="load_gripper" value="false" />
    </include>

    <param name="pose_name" value="$(arg pose_name)" />
    <node name="move_to" pkg="mars_control" type="move_to.py" output="screen" required="true" />
</launch>